assets/js/blocks/rating-filter/block.tsx

```diff
@@ -240,14 +283,39 @@ const RatingFilterBlock = ( {
 				<CheckboxList
 					className={ 'wc-block-rating-filter-list' }
 					options={ displayedOptions }
-					checked={ productRatingsArray }
-					onChange={ ( checked ) => {
-						onClick( checked.toString() );
+					checked={ checked }
+					onChange={ ( item ) => {
+						onClick( item.toString() );
 					} }
 					isLoading={ isLoading }
 					isDisabled={ isDisabled }
 				/>
 			</div>
+			{
+				<div className="wc-block-rating-filter__actions">
+					{ checked.length > 0 && ! isLoading && (
+						<FilterResetButton
+							onClick={ () => {
+								setChecked( [] );
+								setProductRatings( [] );
```

Comment:
Hi @danieldudzic - I recently ran into an error of undefined function `setProductRatings`. Is this a typo and should actually be `setProductRatingsQuery`?

Reply:
@roykho can you please provide some replication steps for this error? :)

Reply:
I've created an issue and opened a PR for this. Good catch @roykho !

https://github.com/woocommerce/woocommerce-blocks/issues/8373